### ISSUE TITLE
vivaldi must be removed from optional packages

### ISSUE THREAD

**Issue by killajoe** [Antergos/Cnchi#1132]
as vivaldi is no longer available on Antergos repository installation will fail if you choose to install optional package vivaldi
;)
![Bildschirmfoto vom 2019-03-18 20-03-04](https://user-images.githubusercontent.com/16797647/54556314-e5d0fe80-49b8-11e9-92a0-10afcef55e20.png)

**Comment by karasu** [comment]
Good catch. I'll remove it asap. I'll close this when it is done.

**Comment by LinuxUserGD** [comment]
Without selecting Vivaldi I still get the error message "Can't download needed packages. Cnchi can't continue.": add_fatal in /usr/share/cnchi/src/misc/events.py:83. It stops while fetching brltty-minimal 6.0-1.
@karasu

**Comment by LinuxUserGD** [comment]
Should I open another issue?

**Comment by killajoe** [comment]
> Should I open another issue?

yes as it is not the same issue better... and as far as i can see brltty-minimal is available on the repo...
Better give the full cnchi log on a new post...

**Comment by killajoe** [comment]
Still present at cnchi and current ISO

**Comment by LinuxUserGD** [comment]
#1151
